src/Cake.Scripting/CodeGen/CakeScriptGenerator.cs

```diff
@@ -122,7 +123,11 @@ public CakeScript Generate(FileChange fileChange)
 
             // Find aliases
             _log.Verbose("Finding aliases...");
-            var aliases = _aliasFinder.FindAliases(references);
+            var aliases = new List<CakeScriptAlias>();
+            foreach (var reference in references)
+            {
+                aliases.AddRange(_aliasFinder.FindAliases(reference));
```

Comment:
And underlying aliasfinder will not return null, it will return empty list instead.